python/arcticdb/util/test.py

```diff
@@ -35,7 +35,7 @@
 # leave out Mongo as spinning up a Mongo instance in Windows CI is fiddly, and Mongo support is only
 # currently required for Linux for internal use.
 # We also skip it on Mac as github actions containers don't work with macos
-RUN_MONGO_TEST = sys.platform == "linux"
```

Comment:
We need to keep running these tests? Man Group uses Mongo as a backend in a few places

Reply:
We will leave it off for now, same as `master` for bypassing the flaky mongo test due to increased no. of CPU cores.
Confirmed with @qc00 that his PR will have the fix for it, along with re-enabling the relevant tests.